### PR TITLE
Sagemaker / Serverless documentation update

### PR DESCRIPTION
**Pull request by yubozhao**
What changes were proposed in this pull request?
------------------------------------------------
Update readme files for deploy with sagemaker / serverless. 

Does this close any currently open issues?
------------------------------------------

How was this patch tested?
--------------------------

### CODE REVIEW

File: examples/deploy-with-sagemaker/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+After deployment, we will show you how to check the deployment status and how to delete this deployment.
+
+### What is AWS SageMaker
+
+AWS SageMaker is a fully-managed service that provide data scientists and developers to train or deploy machine learning models.
+It offer remote Jupyter notebook for building model and help you train and deploy models from the frameworks that they support.
+
+## Prerequisites for deploying model to AWS SageMaker
+
+* And activated AWS account configured on your machine
```
Comment (parano) [inline examples/deploy-with-sagemaker/README.md]: s/And/An

File: examples/deploy-with-sagemaker/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Docker. You can install it on your system with this [Instructions](https://docs.docker.com/install/)
+
+## Deploy to AWS SageMaker with BentoML
+
+After you exported your model with BentoML, you can invoke `bentoml deploy` command for SageMaker deployment. You can invoke that command either inside the Jupyter notebook or inside your terminal.
+
+To invoke command in Jupyter notebook
+
+```
+!bentoml deploy /ARCHIVED_PATH --platform=aws-sagemaker --region=AWS_REGION --api-name=predict
```
Comment (parano) [inline examples/deploy-with-sagemaker/README.md]: maybe just create a similar guide inside a jupyter notebook, for those prefer to read and run this guide from a notebook?

File: examples/deploy-with-sagemaker/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+To invoke command in terminal
```
Comment (parano) [inline examples/deploy-with-sagemaker/README.md]: nit; Update 'ARCHIVED_PATH' to your saved BentoML archive and run the following command in terminal:

File: examples/deploy-with-sagemaker/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+![ScreenShot](./aws-model.png)
+
+![ScreenShot](./aws-endpoint-config.png)
+
+Last, BentoML will create an AWS SageMaker endpoint.
+
+![ScreenShot](./aws-endpoint.png)
+
+## Check deployment status
```
Comment (parano) [inline examples/deploy-with-sagemaker/README.md]: add curl request

File: examples/deploy-with-sagemaker/deploy-with-sagemaker.ipynb [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Check deployment status\n",
+    "\n",
+    "```\n",
+    "bentoml checkt-deployment-status ARCHIVE_PATH --platform=PLATFORM --region=REGION --stage=STAGE\n",
```
Comment (parano) [inline examples/deploy-with-sagemaker/deploy-with-sagemaker.ipynb]: similarly in the notebook,  add curl request example

File: examples/deploy-with-sagemaker/README.md (hunk 1 above)
Comment (yubozhao) [reply examples/deploy-with-sagemaker/README.md]: done

File: examples/deploy-with-sagemaker/README.md (hunk 3 above)
Comment (yubozhao) [reply examples/deploy-with-sagemaker/README.md]: done

File: examples/deploy-with-sagemaker/README.md (hunk 4 above)
Comment (yubozhao) [reply examples/deploy-with-sagemaker/README.md]: done

File: examples/deploy-with-sagemaker/deploy-with-sagemaker.ipynb (hunk 5 above)
Comment (yubozhao) [reply examples/deploy-with-sagemaker/deploy-with-sagemaker.ipynb]: done

File: examples/deploy-with-sagemaker/output.json [hunk 6]
```diff
@@ -0,0 +1 @@
+[4, 0, 0, 4]
```
Comment (parano) [inline examples/deploy-with-sagemaker/output.json]: add this file to gitignore as well?

**Pull request merged**